gensrc/thrift/InternalService.thrift

```diff
@@ -368,64 +370,6 @@ struct TTransmitDataResult {
   4: optional Types.TPlanNodeId dest_node_id
 }
 
-struct TTabletWithPartition {
```

Comment:
What are these deleted data structures for? Will there be compatibility issues during rolling upgrades?

Reply:
TTabletWithPartition only use in `tablet_sink.cpp`, it doesn't used crossed node, so it doesn't has compatibility issue.